### ISSUE TITLE
Cannot read property 'config' of undefined

### ISSUE THREAD

**Issue by federico-hv** [HaikuTeam/cli#1]
Hi, first of all thanks for this awesome animation tool. So, I was trying to export a simple project I have been working on to use it with react and after running `npm install -g @haiku/cli` which installs version `4.5.1` I was trying to run `haiku init` following the steps but I kept getting the following error:

```
sdk_client_1.client.config.getenv();
                           ^

TypeError: Cannot read property 'config' of undefined
```

I have tried with different `node` versions (from 6 to 11) but it is still happening, so I had to specify version `4.5.0` of haiku cli when installing (`npm install -g @haiku/cli@4.5.0`) and that one is working fine. Just to let you know there might be an issue with the latest changes on the cli.

**Comment by stristr** [comment]
Thanks so much for flagging to our attention, @federico-hv. It looks like our last NPM publish of a downstream dependency (`@haiku/sdk-client`) went bad. We'll get this patched up ASAP.

**Comment by stristr** [comment]
Thanks again @federico-hv. You should be able to pop over to our latest version without issue, published at version `4.5.2`. Please let us know if you still see problems on the very latest.

**Comment by federico-hv** [comment]
Version `4.5.2` is working fine. Thanks @stristr
